app/_data/extensions/kong-inc/kafka-upstream/versions.yml

```diff
@@ -1 +1 @@
-- release: 1.3-x
+- release: 1.3-x (internal 0.2.0)
```

Comment:
same question from https://github.com/Kong/docs.konghq.com/pull/2595/#discussion_r578373044 applies

Reply:
> same question from #2595 applies

this was a POC but I'm removing this from the PR. The focus of this PR is to add the datatypes. The plugin versions are another lomgstanding matter that needs product management input on how to handle going forward.